types/react-linkify/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-linkify 0.2
+// Type definitions for react-linkify 1.0
```

Comment:
LGTM

Reply:
Looks like the stable version of react linkify is still in alpha... should we update the version here to reflect that? https://www.npmjs.com/package/react-linkify

Reply:
There is a linter rule which doesn't let you set a patch version. https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md